ssl-config/src/main/java/com/palantir/remoting/ssl/SslConfiguration.java

```diff
@@ -28,18 +30,44 @@
 @Value.Style(visibility = ImplementationVisibility.PACKAGE)
 public abstract class SslConfiguration {
 
-    @Value.Parameter
-    public abstract TrustStoreConfiguration trust();
+    public abstract URI trustStorePath();
```

Comment:
These are different from the Dropwizard spec in that they take URIs and not paths. Is this still actually a reasonable name?